tests/flytekit/integration/remote/test_remote.py

```diff
@@ -591,6 +592,135 @@ def test_flyteremote_uploads_large_file(gigabytes):
             assert s3_md5_bytes == md5_bytes
 
 
+@mock.patch("flytekit.tools.interactive.ipython_check")
+def test_workflow_remote_func(mock_ipython_check):
+    """Test the logic of the remote execution of workflows and tasks."""
+    mock_ipython_check.return_value = True
```

Comment:
Should we set `interactive_mode_enabled` to True for the remote client instead?

Reply:
You are right